client/src/typings/common/IUser.ts

```diff
@@ -12,4 +12,9 @@ export interface IUser {
 	clan?: WebApi.Entities.IClan;
 	profileUrl?: string;
 	github?: IGithubProfile;
+	email?: string;
+	skills?: Array<string>;
+	devLevel?: string;
+	social?: Array<string>;
+	avatar?: string;
```

Comment:
`avatar` and `profileUrl` are the same fields, so remove `profileUrl`

Reply:
This will be a separate branch, because we will need to review and fix all the other components, as there are many uses of "profileUrl" and many uses of "avatar"